R/save.R

```diff
@@ -102,7 +102,7 @@ ggsave <- function(filename, plot = get_last_plot(),
   dim <- plot_dim(c(width, height), scale = scale, units = units,
     limitsize = limitsize, dpi = dpi)
 
-  if (is_null(bg)) {
+  if (is_null(bg) && is.ggplot(plot)) {
```

Comment:
In the case a list is passed in, I think we should just grab the bg from the first plot in the list rather than completely ignore it

Reply:
This is sensible thanks!